### ISSUE TITLE
Revise implementation of ManualColumnResize, position misplaced when there is a transform on container.

### ISSUE THREAD

**Issue by DongShi** [handsontable/handsontable#5050]
### Description

By looking at the css for the [column resizer](https://github.com/handsontable/handsontable/blob/7476f640754c49cd395d2c966a161540802c95b8/src/css/handsontable.css#L178). You can see that "Position: fixed" is used to locate the position of the resizer dom. 
However, there is a long discussed issue about ['CSS3 transform and fixed position'](https://coderwall.com/p/ka0low/don-t-mix-position-fixed-with-css-transforms) and a chrome [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=20574&desc=2) tracking this. 

So if you put HOT in a container inside a container with transform defined, you probably will find your resizer misplaced.
because the coordinate space is kind of "hijacked" by the transform.

This strange behavior will impact us a lot since we are using libraries like [react-grid-layout](https://github.com/STRML/react-grid-layout/tree/master/examples), which use transform pretty much.

Not sure if the same problem applied to row resizer...

### Steps to reproduce

1. Just put your HOT instance inside a container node with transform set.
2.
3.

### Demo

https://jsfiddle.net/9qs86ee4/

### Your environment
* Handsontable version: Latest
* Browser Name and version: Chrome 64
* Operating System: MacOS

**Comment by AMBudnik** [comment]
Hi @DongShi 

Thank you for reporting it. I believe that we have mentioned this subject at https://github.com/handsontable/handsontable/issues/3946 and https://github.com/handsontable/handsontable/issues/4394 but you've gathered a bunch of useful information so that will definitely help.

**Comment by DongShi** [comment]
hello @AMBudnik is there any progress on this or related issues?  We are blocked by this to apply HC in our production.

Let me know  if there is any news.

**Comment by wojciechczerniak** [comment]
No news. But with the links you've provided we have a solid research into the issue already done. They're not going to fix this in the browsers, we have to adapt. I will add this to the next sprint backlog.

Note: Both `manualColumnResize` and `manualRowResize` are affected. We should check `manualRowMove` and `manualColumnMove` either.

For reference:

`transform: scale()`: #3946
`transform: translate()`: #4394 (react-grid-layout related)
`transform: translate()`: #2937 (bootstrap modal + react-grid-layout related)
`transform: translate()`: #3647 (bootstrap modal)

Workaround/fix idea: https://github.com/handsontable/handsontable/issues/2937#issuecomment-356601813

**Comment by vmagalhaes** [comment]
Any update? I'm having this bug on bootstrap modal and I'm using Handsontable Angular

**Comment by AMBudnik** [comment]
Hey @vmagalhaes 

the issue has a milestone attached. We are working on a big change in Handsontable write now so there are no new commits, but we are planning to schedule some work for this issue in February

**Comment by AMBudnik** [comment]
Hey @DongShi 

we've just released a new version of Handsontable 7.0.1 where this issue is fixed.

Thank you for posting!

**Comment by howoocast** [comment]
wow! this issue is finally fixed! thanks a lot!

**Comment by AMBudnik** [comment]
Thanks for feedback @howoocast :)

**Comment by DongShi** [comment]
Good work， cant wait to try this fix out!
